src/types/tables.ts

```diff
@@ -21,16 +23,20 @@ export interface FormattedData {
 
 export type DataIndex = keyof DataTableDataType;
 
-export interface PublicationObjectDataItem {
-  ApplicableAt: string;
-  ApplicableFor: string;
-  Value: number;
-  GeneratedTimeStamp: string;
-  QualityIndicator: string;
-  Substituted: string;
-  CreatedDate: string;
+export enum InstantaneousFlowTableHeaderName {
+  systemEntry = 'System Entry Name',
+  terminalTotals = 'Terminal Totals',
+  totalSupply = 'Total System Supply',
+  interconnectorFlow = 'Interconnector Export Totals',
+  totalDemand = 'Total Demand Data',
+  actualLinepack = 'NTS Actual Linepack',
 }
-export interface DataItemExplorerDataItem {
-  PublicationObjectName: string;
-  PublicationObjectData: { CLSPublicationObjectDataBE: PublicationObjectDataItem[] };
+
+export interface InstantaneousFlowPageData {
```

Comment:
We already have this type: DataApiResponse that has gasDay and data. 

I would reuse e.g. see export type LatestSupplyEntryPointResponseData = DataApiResponse<SystemEntryPointData>;

Reply:
The only thing is, is that this is the transformed data, not the API reponse, so might be misleading?

Reply:
I'll do it once all merged and we're in the repo (I won't forget don't worry)